libraries/AP_Mission/examples/AP_Mission_test/AP_Mission_test.cpp

```diff
@@ -15,8 +15,8 @@ class MissionTest {
 
 private:
     AP_InertialSensor ins;
-    AP_Baro baro;
     AP_GPS  gps;
+    AP_Baro baro;
```

Comment:
Noise/accident? (I suspect it's left over from resolving rebases/singleton changes)

Reply:
Yes, leftover